scripts/Release.sh

```diff
@@ -0,0 +1,186 @@
+CurrentVersion=$(sed -n '/s.version/{s/s.version//;s/=//;s/[\"]//g;s/^[[:space:]]*//g;p;q;}' $PodSpecFile)
+echo "Current Version="$CurrentVersion
+
+# 2.2) prompt user for new version
+echo "New Version?"
+read NewVersion
+
+# 2.3) if not blank, or not the same
+if [ -z $NewVersion ]; then
+    echo No file change needed
+else
+    if [ $CurrentVersion != $NewVersion ]; then
+        echo make the change
+        # 2.4) swap new version in to file
+        sed '/s.version/{s/'$CurrentVersion'/'$NewVersion'/;}' $PodSpecFile > $PodSpecNewFile
+        mv -f $PodSpecNewFile $PodSpecFile
+    else
+        echo No file change needed for version
+    fi
+fi
+
+
+# 3 update RPC versions
+# file?
+
+# 3 update protocol versions
+# file?
+
+# 4 update to newest bson
+# file?
+
+# 4 update package.swift
+# ?
+
+# 4 update CocoaPods dependancy
+# ?
+
+# 5 update changelog
+# ?
+
+# 6 Install Jazzy
+echo "6 Install Jazzy"
+sudo gem install jazzy
+
+# 6 generate documentation
+echo "6 generate documentation"
+#bash generate-documentation.sh
+
+
+# 7 ensure RPC_SPEC has released to master
+
+# 7 update the submodule to point to new release
+
+
+# git commands
+# 8 commit release to develop
+# git checkout <branch>
+# git add -A
+# git commit -m <message here>
+# git push --set-upstream origin <branchname>
+
+# 8 merge release to master
+# git merge <branch> <master>
+
+# 8 tag it
+# git tag <something>
+
+# 8 merge master back to develop
+# git merge <branch> <master>
+
+
+
+# 1 Create new release for tag
+# ?
+
+# 1 add highlights of changes
+# ?
+
+
+# 2 push new release to primary cocoapod
+echo "2 push new release to primary Cocoapod"
+#pod trunk push SmartDeviceLink.podspec --allow-warnings
+
+# 3 Push the new release to the secondary cocoapod using command line:
+echo "3 Push the new release to the secondary cocoapod using command line:"
+#pod trunk push SmartDeviceLink-iOS.podspec --allow-warnings.
+
+# 4 add binary xcframework archive for manual installation
+echo "4 add binary xcframework archive for manual installation"
+# i
+echo "4.i"
+xcodebuild archive -project 'SmartDeviceLink-iOS.xcodeproj/' -scheme 'SmartDeviceLink' -configuration Release -destination 'generic/platform=iOS' -archivePath './SmartDeviceLink-Device.xcarchive' SKIP_INSTALL=NO
+
+# ii
+echo "4.ii"
+xcodebuild archive -project 'SmartDeviceLink-iOS.xcodeproj/' -scheme 'SmartDeviceLink' -configuration Release -destination 'generic/platform=iOS Simulator' -archivePath './SmartDeviceLink-Simulator.xcarchive' SKIP_INSTALL=NO
+
+# iii
+echo "4.iii"
+xcodebuild -create-xcframework -framework './SmartDeviceLink-Device.xcarchive/Products/Library/Frameworks/SmartDeviceLink.framework/' -framework './SmartDeviceLink-Simulator.xcarchive/Products/Library/Frameworks/SmartDeviceLink.framework/' -output './SmartDeviceLink.xcframework'
+
+# iv Compress the .xcframework and add the it to the release.
+echo "4.iv Compress the .xcframework and add the it to the release."
```

Comment:
Zip the `.xcframework` and name it based on the naming scheme in the releases. Then delete the old intermediate files.

Reply:
Not completed yet.  Commands are in place.